### ISSUE TITLE
[Bug]: Fireworks broken on 1.34.0. Fine on 1.33.1

### ISSUE THREAD

**Issue by mchal** [tsparticles/tsparticles#2140]
### What happened?

I was checking the Fireworks demo on codepen made by @matteobruni 
https://codepen.io/matteobruni/pen/abJQrbK

and it looks fine for tsparticle 1.33.1, 
![image](https://user-images.githubusercontent.com/327465/132029703-c1714305-5a12-496e-8afa-28661a88b8f5.png)

but once I change it to 1.34.0 ( using https://cdn.jsdelivr.net/npm/tsparticles@1.34.0/tsparticles.min.js ) fireworks are very off (no "shooting" line, fireworks are going in random direction)
![image](https://user-images.githubusercontent.com/327465/132029507-29f259cd-e928-4959-91b8-37fd3220e3f7.png)

Nothing else has been changed

### Version

1.34.0

### Which library are you using?

Vanilla (tsparticles)

### tsParticles Configuration

```json
https://codepen.io/matteobruni/pen/abJQrbK

Same config; for 1.33.1 looks fine, 1.34.0 is wrong
```

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by matteobruni** [comment]
I fixed the demo to work with 1.34, the 1.33 has a size value bug fixed in 1.34 that breaks the old fireworks config

**Comment by mchal** [comment]
Thank you.
Can confirm this is resolve it now (and fireworks on my website as well :))
